composer.json

```
@@ -44,6 +44,8 @@
         "phpstan": "vendor/bin/phpstan analyse --ansi --error-format symplify",
         "check-cs": "vendor/bin/ecs check --ansi",
         "fix-cs": "vendor/bin/ecs check --fix --ansi",
+        "check-rector": "vendor/bin/rector process --dry-run --ansi",
+        "fix-rector": "vendor/bin/rector process --ansi",
```

to sync with what applied in rector-src:

```suggestion
        "rector": "vendor/bin/rector process --ansi",
```

---

Done ;)